src/openrct2-ui/UiContext.macOS.mm

```diff
@@ -118,14 +118,14 @@ void ShowMessageBox(SDL_Window * window, const std::string &message) override
                 panel.canChooseFiles = false;
                 panel.canChooseDirectories = true;
                 panel.allowsMultipleSelection = false;
-                utf8 *url = NULL;
                 if ([panel runModal] == NSFileHandlingPanelOKButton)
                 {
                     NSString *selectedPath = panel.URL.path;
                     const char *path = selectedPath.UTF8String;
```

Comment:
This function is supposed to return an `std::string`, so shouldn't `path` be changed into that? (I know that const char is C's equivalent, but I'm not sure if they're interchangeable.)